### ISSUE TITLE
'EACCES: permission denied, open '/var/www/ghost/.ghost-cli

### ISSUE THREAD

**Issue by nmaxcom** [TryGhost/Ghost-CLI#485]
```
ubuntu@xx:/var/www/ghost$ ghost start
✔ Validating config
Running sudo command: systemctl start ghost_xx-com
✖ Starting Ghost
An error occurred.
Message: 'EACCES: permission denied, open '/var/www/ghost/.ghost-cli''
```

This happened after a successful `ghost run` to check everything (after a hundred bugs, reinstalls, and other issues). I think I had this error before and a knowledgeable guy from Slack told me I needed to add one more key to `.ghost-cli`. Something like "environment":"production" or something like that. He mentioned it was a known bug but I didn't find it here.

I'm asking here so if it happens to anyone else or me again I'll have somewhere to search. Also, I already "patched" that filed but somehow it's just a three liner like it was before:

```
{
  "cli-version": "1.1.1",
  "active-version": "1.10.0",
  "name": "xx-com"
}
```

I'm sorry if I sound irritated, but I don't understand, from my perspective, why on earth such a buggy product was released. I'm afraid it'll break mid-fly.

### Technical details (will be automatically output by Ghost-CLI if an error occurs):

* Linux 4.4.0-1022-aws #31-Ubuntu
* Node Version: v6.11.3
* Ghost-CLI Version: 1.1.1
* Environment: production

**Comment by ErisDS** [comment]
@nmaxcom sorry to hear you're having trouble. 

'EACCES: permission denied, open '/var/www/ghost/.ghost-cli'

This means that the ownership or permissions on the mentioned file do not allow for the user being used to run the command in question to open the file.

> after a hundred bugs, reinstalls, and other issues

Unfortunately, without details, it's pretty hard for us to tie down why this situation happened for you.

> I'm sorry if I sound irritated, but I don't understand, from my perspective, why on earth such a buggy product was released. I'm afraid it'll break mid-fly.

No one at Ghost wants you to have a bad experience, that's why we are creating tooling to ease the install process & overcome the enumerable combinations and complexities of a linux setup. Raising a bug without enough information and bemoaning how buggy the software is does nothing but contribute bad feeling whilst hamstringing us from actually fixing the issue.

**Comment by nmaxcom** [comment]
> Unfortunately, without details, it's pretty hard for us to tie down why this situation happened for you.

I sent the details to two teammates. And today I talked with the ghost-cli dev in your Slack. I'm a developer myself; I understand the challenges. **IMHO ghost-cli is not ready even for beta**, and AFAIK is a "stable" solution you offer on your website as the easy-sure-shoot way.

> No one at Ghost wants you to have a bad experience, that's why we are creating tooling to ease the install process & overcome the innumerable combinations and complexities of a linux setup. Raising a bug without enough information and bemoaning how buggy the software is does nothing but contribute bad feeling whilst hamstringing us from actually fixing the issue.

I followed your official install process to the letter, and I found myself with a broken install, even though it was a clean spun Ubuntu 16, as requested.
I had to follow the source of the ghost-cli intentions to manually get everything straight. The mysql mess, nginx etc. There are still a few weird issues though. I think, as of right now, the product is too frail.

And don't get me started with migrating my carefully crafted pre-1.0 theme. Still, *that*, I understand, painful as it is.

I hope you keep progressing, honestly. Ghost is a great platform; I wish I had more time to put my own hours into it.
